src/com/mateakademia/homework/homework6/FlowerStore.java

```diff
@@ -0,0 +1,60 @@
+package com.mateakademia.homework6;
+
+
+public class FlowerStore {
+
+    private int walletOfShop;
+
+   public Flowers[] sellSequence(int rose, int chamomile, int tulip ) {
+
+        int QuantityFlowers = rose + chamomile + tulip;
+        Flowers[] buket = new Flowers[QuantityFlowers];
+        int k = 0;
+        while (k < QuantityFlowers) {
+            if (rose > 0) {
+                buket[k] = new Rose();
+                rose--;
+                k++;
+            }
+            if (chamomile > 0) {
+                buket[k] = new Chamomile();
+                chamomile--;
+                k++;
+            }
+
+            if (tulip > 0) {
+               buket[k] = new Tulip();
+                tulip--;
+                k++;
+            }
+
+        }
+        totalBalance(buket);
+        return buket;
+    }
+
+    private void totalBalance(Flowers[] buket) {
```

Comment:
private method write after public ones
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)

Reply:
and [Method names are typically verbs or verb phrases](https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names)